tests/framework/framework.go

```diff
@@ -45,6 +49,17 @@ func getBackupNamespaceFromEnv() string {
 	return backupNamespace
 }
 
+func getRegionFromEnv() string {
+	region := os.Getenv(regionEnv)
+	if region == "" {
+		fmt.Fprintf(os.Stderr, "defaulting to minio ns\n")
```

Comment:
Consider using a log wrapper func to replace all `fmt.Fprintf(os.Stderr,...)` calls

Reply:
Good Idea, I'll do it in next PR, I really want to test the velero plugin today on some other environments, and this PR will help to make it happed.